Website/Sources/ConferenceWebsite.swift

```diff
@@ -61,6 +62,36 @@ struct ConferenceWebsite {
       }
     }
   }
+
+  private static func copyAASAFile() throws {
+    let fileManager = FileManager.default
+
+    let websiteDirectory = try URL.selectDirectories(from: #file).source
+    let websiteWellKnownDirectory = websiteDirectory.appending(path: ".well-known")
```

Comment:
When using Ignite to manage assets, the file needs to be included in the `Assets` directory.
ref: https://github.com/twostraws/Ignite?tab=readme-ov-file#folder-structure

```suggestion
    let websiteWellKnownDirectory = websiteDirectory.appending(path: "Assets/.well-known")
```

I’ve confirmed that this code resolves the issue.
```sh
❯ curl http://localhost:8000/.well-known/apple-app-site-association
{
    "appclips": {
        "apps": ["jp.tryswift.Clip"]
    }
}%
```